assets/js/components/surveys/CurrentSurvey.test.js

```diff
@@ -121,7 +121,6 @@ describe( 'CurrentSurvey', () => {
 				registry,
 			} );
 
-			// Enter answer text.
```

Comment:
Makes sense. It's readable in the code now :joy: 

These started off as actual TODO notes for what I wanted to do.

Another way would be having explicitly named helper methods e.g `changeInputField()` like KCD does in https://kentcdodds.com/blog/avoid-nesting-when-youre-testing :smile: